commit 3ef8ce179937d5c08ddd4faed7e63ff6cd4a9da8
Author: Madanie <madi@localhost.localdomain>
Date:   Tue Nov 14 13:00:28 2023 +0200

    Made the top bar be half height to reduce wasted space

diff --git a/config.def.h b/config.def.h
index 061ad66..5cb7707 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,6 +5,7 @@ static const unsigned int borderpx  = 1;        /* border pixel of windows */
 static const unsigned int snap      = 32;       /* snap pixel */
 static const int showbar            = 1;        /* 0 means no bar */
 static const int topbar             = 1;        /* 0 means bottom bar */
+static const int shortbar           = 1;        /* 0 means normal bar */
 static const char *fonts[]          = { "monospace:size=10" };
 static const char dmenufont[]       = "monospace:size=10";
 static const char col_gray1[]       = "#222222";
diff --git a/dwm.c b/dwm.c
index e5efb6a..d374126 100644
--- a/dwm.c
+++ b/dwm.c
@@ -1546,7 +1546,7 @@ setup(void)
 	if (!drw_fontset_create(drw, fonts, LENGTH(fonts)))
 		die("no fonts could be loaded.");
 	lrpad = drw->fonts->h;
-	bh = drw->fonts->h + 2;
+   bh = shortbar ? drw->fonts->h * .5 : drw->fonts->h + 2;
 	updategeom();
 	/* init atoms */
 	utf8string = XInternAtom(dpy, "UTF8_STRING", False);
